CI fails: CerebrasException - Missing credentials

CI fails: https://github.com/huggingface/smolagents/actions/runs/25857903395/job/75980209998?pr=2273
> AssertionError: Error message '<ExceptionInfo litellm.InternalServerError: InternalServerError: CerebrasException - Missing credentials. Please pass an `api_key`, `workload_identity`, `admin_api_key`, or set the `CEREBRAS_API_KEY` or `CEREBRAS_ADMIN_KEY` environment variable. tblen=8>' does not contain any expected phrases

```python
FAILED tests/test_models.py::TestLiteLLMModel::test_call_different_providers_without_key[cerebras/llama-3.3-70b] - AssertionError: Error message '<ExceptionInfo litellm.InternalServerError: InternalServerError: CerebrasException - Missing credentials. Please pass an `api_key`, `workload_identity`, `admin_api_key`, or set the `CEREBRAS_API_KEY` or `CEREBRAS_ADMIN_KEY` environment variable. tblen=8>' does not contain any expected phrases
assert False
 +  where False = any(<generator object TestLiteLLMModel.test_call_different_providers_without_key.<locals>.<genexpr> at 0x7feb3a809150>)
```

Stacktrace:
```python
 _ TestLiteLLMModel.test_call_different_providers_without_key[cerebras/llama-3.3-70b] _

self = <tests.test_models.TestLiteLLMModel object at 0x7fec9e345040>
model_id = 'cerebras/llama-3.3-70b'

    @pytest.mark.parametrize(
        "model_id",
        [
            "groq/llama-3.3-70b",
            "cerebras/llama-3.3-70b",
            "mistral/mistral-tiny",
        ],
    )
    def test_call_different_providers_without_key(self, model_id):
        # Different litellm versions produce different error messages for missing API keys
        # This test checks for the presence of any common authentication-related error phrases
        possible_error_messages = [
            "Missing API Key",
            "Wrong API Key",
            "Invalid API Key",
            "The api_key client option must be set",
            "AuthenticationError",
            "Unauthorized",
        ]
        model = LiteLLMModel(model_id=model_id)
        messages = [ChatMessage(role=MessageRole.USER, content=[{"type": "text", "text": "Test message"}])]
        # Test generate method
        with pytest.raises(Exception) as e:
            model.generate(messages)
        error_message = str(e)
>       assert any(possible_error_message in error_message for possible_error_message in possible_error_messages), (
            f"Error message '{error_message}' does not contain any expected phrases"
        )
E       AssertionError: Error message '<ExceptionInfo litellm.InternalServerError: InternalServerError: CerebrasException - Missing credentials. Please pass an `api_key`, `workload_identity`, `admin_api_key`, or set the `CEREBRAS_API_KEY` or `CEREBRAS_ADMIN_KEY` environment variable. tblen=8>' does not contain any expected phrases
E       assert False
E        +  where False = any(<generator object TestLiteLLMModel.test_call_different_providers_without_key.<locals>.<genexpr> at 0x7feb3a809150>)

tests/test_models.py:417: AssertionError
```

--- a/tests/test_models.py
+++ b/tests/test_models.py
@@ -407,6 +407,7 @@ class TestLiteLLMModel:
             "The api_key client option must be set",
             "AuthenticationError",
             "Unauthorized",
+            "Missing credentials",
         ]
         model = LiteLLMModel(model_id=model_id)
         messages = [ChatMessage(role=MessageRole.USER, content=[{"type": "text", "text": "Test message"}])]

